### DISCUSSION TITLE
Not found when first query param is matches some of provided locale

### DISCUSSION THREAD

**Discussion by danyasnov** [vercel/next.js#26676] [Help]
I want to use routing like `[country]/[region]/[category]` e.g. `/RU/saint-petersburg/property`
I also have a list of locales which includes `RU`

so then I go `/RU/saint-petersburg/property` I got `404 This page could not be found.` because next.js implicitly uses the first parameter as the language setting which is not required since I am using domain based i18n

How can I avoid such behavior?

I'm using latest next.js 11.0.1

**Comment by sdim2** [comment]
Same problem
